charts/opentelemetry-collector/ci/metrics-values.yaml

```diff
@@ -0,0 +1,8 @@
+ports:
+  metrics:
+    enabled: true
```

Comment:
Not sure we need this one. This port needed for PodMonitor and ServiceMonitor only which we decided to not cover here

Reply:
Fair, only testing that the metric port can be enabled isn't very substantial.  Removed.